src/storage/metrics.rs

```diff
@@ -14,7 +14,7 @@
 use prometheus::*;
 
 lazy_static! {
-    pub static ref KV_COMMAND_COUNTER_VEC: IntCounterVec =
+    pub static ref COMMAND_COUNTER_VEC: IntCounterVec =
```

Comment:
I think we can still name it `KV_COMMAND_COUNTER_VEC`, just merge `RAWKV_COMMAND_COUNTER_VEC` into it.